public/i18n/en/translation.json

```
@@ -976,7 +976,9 @@
       "interface_not_found": "Interface not found",
       "no_dynamic_lease_found": "No dynamic leases found",
       "filter_change_suggestion": "Try changing filter value",
-      "dhcp_option_combobox_placeholder": "Search key"
+      "dhcp_option_combobox_placeholder": "Search key",
+      "force_start": "Force DHCP server start",
+      "force_start_tooltip": "When force option is disabled, the DHCP will not start if there is another DHCP in the network. Disable the force option, only if you want to avoid conflicts with other DHCP servers."
```

Maybe it' more clear using positive logic:
```suggestion
      "force_start_tooltip": "When force option is enabled, DHCP will start even if there is another DHCP server in the network. Disable force option only if you want to avoid conflicts with other DHCP servers."
```